.github/workflows/ci.yml

```diff
@@ -13,6 +13,9 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
Just noticed this and the thought "should we be using Debian instead?" popped up?

Reply:
anecdotally, i would say ubuntu seems to be more standard and common for things like this. personally, i tend to think "ubuntu unless i have a reason not to"

Reply:
Yeah, that used to be the previous default for a lot of people, though it doesn't seem to be the case as much any more.

For me, when Canonical started including advertising in their cli is when I personally stopped defaulting to Ubuntu for anything (now defaulting to Debian instead).

For an existing build process though (like here), it's unlikely to be worth the effort to go about changing the distribution due to trivial preference stuff like that.  So yeah, no worries. :wink: